### PR TITLE
config: test jdk12

### PR DESCRIPTION
**Pull request by romani**

### CODE REVIEW

File: .ci/travis/travis.sh [hunk 1]
```diff
@@ -116,7 +116,7 @@ jdk12)
   exclude3="\!TokenUtilTest#testTokenValueIncorrect2,"
   exclude4="\!ImportControlLoaderPowerTest#testInputStreamThatFailsOnClose"
 
-  mvn -e package -Passembly -Dtest=$exclude1$exclude2$exclude3$exclude4
+  mvn -e package -Passembly
```
Comment (rnveach) [inline .ci/travis/travis.sh]: https://github.com/checkstyle/checkstyle/blob/master/pom.xml#L1665
seems tests are skipped with the assembly profile.

Comment (romani) [reply .ci/travis/travis.sh]: Strange that build is green , even it are failing https://travis-ci.org/checkstyle/checkstyle/jobs/597127920#L1353

Comment (rnveach) [reply .ci/travis/travis.sh]: It seems to me we have seen something like this with JDK 12 before.

Comment (romani) [reply .ci/travis/travis.sh]: We simply have mistakes in build commands, https://github.com/checkstyle/checkstyle/pull/7192 will fix them.

**Pull request closed without merging**